src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -652,6 +652,16 @@ The following fields are available to configure your Azure infrastructure.
 - *domain:* Corresponds to the Azure _domain_, _tenant ID_, or _directory ID_ of your Active Directory.
 - *subscriptionId:* The _ID_ of your Azure subscription.
 
+The next parameters of the Azure Billing Configuration section, allow you to configure the automatic cloud cost estimator, by considering all the Azure resources related to your reservation (virtual machines, disks,..). This mechanism relies on the Azure Resource RateCard and Usage APIs (https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/usage-rate-card-overview).
```

Comment:
reservation => allocation

Reply:
ok

Reply:
the Azure Resource RateCard and Usage APIs => the Azure Resource Usage and RateCard APIs (as mentioned in https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/usage-rate-card-overview)

Reply:
ok to keep `reservation ` np :))